vendor/vendor.json

```diff
@@ -155,6 +155,12 @@
 			"revision": "89623f28b87a6004d4b785663257362d1658a729",
 			"revisionTime": "2017-12-05T05:15:20Z"
 		},
+		{
+			"checksumSHA1": "IqqBK06hDnn7N1nd9YT9mwrr96w=",
+			"path": "github.com/containerd/containerd/contrib/seccomp",
```

Comment:
Do we need to vendor containerd/containerd package in a unified version. I found previous, we vendored containerd, using 
```
		{
			"checksumSHA1": "PriKCFVJIjLn3y1jLTNb+9RsG7A=",
			"path": "github.com/containerd/containerd/remotes/docker",
			"revision": "89623f28b87a6004d4b785663257362d1658a729",
			"revisionTime": "2017-12-05T05:15:20Z"
		},
		{
			"checksumSHA1": "CHt96fWy3eXjcCqxJg5SXjkofCA=",
			"path": "github.com/containerd/containerd/remotes/docker/schema1",
			"revision": "89623f28b87a6004d4b785663257362d1658a729",
			"revisionTime": "2017-12-05T05:15:20Z"
		},
		{
			"checksumSHA1": "OuOr/HVV29fKoThNjloWKOb13YE=",
			"path": "github.com/containerd/containerd/rootfs",
			"revision": "89623f28b87a6004d4b785663257362d1658a729",
			"revisionTime": "2017-12-05T05:15:20Z"
		},
```

While here for this package seccomp, we vendored `83cf37155670575ec0782bb7940b53876c151c40 `.

Maybe all packages from same version would be make more sense? WDYT? @YaoZengzeng